src/modules/Hosts/Hosts/MainWindow.xaml.cs

```diff
@@ -16,6 +19,22 @@ public MainWindow()
             SetTitleBar();
 
             BringToForeground();
+
+            Activated += MainWindow_Activated;
+        }
+
+        private void MainWindow_Activated(object sender, WindowActivatedEventArgs args)
+        {
+            if (args.WindowActivationState == WindowActivationState.Deactivated)
+            {
+                AppTitleTextBlock.Foreground =
+                    (SolidColorBrush)App.Current.Resources["WindowCaptionForegroundDisabled"];
+            }
+            else
+            {
+                AppTitleTextBlock.Foreground =
+                    (SolidColorBrush)App.Current.Resources["WindowCaptionForeground"];
```

Comment:
AppTitleTextBlock.Foreground =                     (SolidColorBrush)App.Current.Resources["WindowCaptionForeground"];

Reply:
It's directly copied from a MS example. But if it's important I'll change it...

Reply:
Not important at all, just a personal preference since I not a fan of new lines where aren't really needed.